addressableled/robot.py

```diff
@@ -0,0 +1,51 @@
+import wpilib
+
+kLEDBuffer = 60
+
+
+class MyRobot(wpilib.TimedRobot):
+    def robotInit(self):
+        # PWM Port 9
+        # Must be a PWM header, not MXP or DIO
+        self.led = wpilib.AddressableLED(9)
+
+        # LED Data
+        self.ledData = [wpilib.AddressableLED.LEDData() for i in range(kLEDBuffer)]
+
+        # Store what the last hue of the first pixel is
+        self.rainbowFirstPixelHue = 0
+
+        # Default to a length of 60, start empty output
+        # Length is expensive to set, so only set it once, then just update data
+        self.led.setLength(kLEDBuffer)
+
+        # Set the data
+        self.led.setData(self.ledData)
+        self.led.start()
+
+    def robotPeriodic(self):
+        # Fill the buffer with a rainbow
+        self.rainbow()
+
+        # Set the LEDs
+        self.led.setData(self.ledData)
+
+    def rainbow(self):
+        # For every pixel
+        for i in range(kLEDBuffer):
```

Comment:
Thoughts on using the more Pythonic `enumerate`, @virtuald? Or would that perhaps be too confusing?

Reply:
This is fine.